test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -10546,13 +10546,18 @@ def forward(self, x):
     #       such that inputs and outputs do not always overflow/underflow.
     #       Otherwise test results could be inaccurate.
     @skipIfUnsupportedMinOpsetVersion(10)
-    @unittest.skip("https://github.com/pytorch/pytorch/issues/74501")
     def test_quantized_linear(self):
         model = torch.nn.quantized.Linear(4, 8)
+        # Set fixed weight to avoid flaky test.
+        weight = torch.quantize_per_tensor(
+            torch.arange(32).view(8, 4).to(torch.float),
```

Comment:
Is `.to(torch.float)` needed? I thought that was the default for these tests?

if it is needed, wouldn't setting `dtype=torch.float` in the call to `arange` be more efficient?

Reply:
yes it's needed, arange default returns int64. I will update with the dtype suggestion.